### DISCUSSION TITLE
EJS language support extension not working

### DISCUSSION THREAD

**Discussion by EddiesTech** [coder/code-server#2750] [Q&A]
Hi there.
The extension "EJS language support" works fine on my normal VS Code, but doesn't work here. I get these errors:
> [DigitalBrainstem.javascript-ejs-support]: Cannot register configuration defaults for 'emmet.includeLanguages'. Only defaults for language specific settings are supported.

and

> [DigitalBrainstem.javascript-ejs-support]: Cannot register configuration defaults for 'files.associations'. Only defaults for language specific settings are supported.

This seems to only be a problem on `code-server` as it works on my normal VS Code. I tried changing the settings myself to what the extension changes them to, but the errors still come up and there is no highlighting. Any ideas on how to fix this?

**Comment by EddiesTech** [comment]
Should probably give links
VS Code Marketplace: https://marketplace.visualstudio.com/items?itemName=DigitalBrainstem.javascript-ejs-support
Open VSX Registry: https://open-vsx.org/extension/DigitalBrainstem/javascript-ejs-support

**Comment by Yukigamine** [comment]
I'm seeing the same issue. Did you ever figure anything out?

**Comment by EddiesTech** [comment]
From what I can see, the error means nothing and the highlighting actually works fine. The highlighting works for me, I just didn't realise it at the time.

**Reply by jsjoeio** [reply]
well, that's good I guess! I'm not sure what we'd have changed in code-server to cause this. Do you mind posting steps to reproduce? I can look
